oio/api/object_storage.py

```diff
@@ -411,6 +411,11 @@ def container_snapshot(self, account, container, dst_account,
             self.container.container_freeze(account, container, **kwargs)
             self.container.container_snapshot(
                 account, container, dst_account, dst_container, **kwargs)
+            # TODO(mbo) maybe it should be done in proxy
+            self.container.container_set_properties(
+                dst_account, dst_container,
+                system={'sys.user.name': dst_container,
+                        'sys.account': dst_account})
```

Comment:
If the snapshot is supposed to be a "real" container, and not just a snapshot, then yes, this should be done by oio-proxy. I thought for one moment that this could have been done even one level below, but since this uses an sqliterepo feature, there is no notion of container or account (`sys.account` and `sys.user.name` are meta2-only entries).